crates/storage/db/src/abstraction/mock.rs

```diff
@@ -105,31 +105,42 @@ pub struct CursorMock {
 
 impl<T: Table> DbCursorRO<T> for CursorMock {
     fn first(&mut self) -> PairResult<T> {
-        todo!()
+        self._cursor = 0;
+        Ok(None)
```

Comment:
I'm okay with returning None everywhere,

but let's remove the cursor updates, don't think we need them

Reply:
> I'm okay with returning None everywhere,
> 
> but let's remove the cursor updates, don't think we need them

ty lot Mat!